pkg/controller/direct/logging/utils.go

```diff
@@ -285,7 +285,6 @@ func convertKCCtoAPIForMetricDescriptor(kccObj *krm.LogmetricMetricDescriptor) *
 	metricDescriptor := &api.MetricDescriptor{}
 	if kccObj.DisplayName != nil {
 		metricDescriptor.DisplayName = *kccObj.DisplayName
-		metricDescriptor.Name = *kccObj.DisplayName
```

Comment:
Hm, I recall adding this for backwards compat reasons but I don't see these tests failing so maybe I got confused 🤷🏼 

good catch!

Reply:
I think we shall not override the name anyways because `metricDescriptor.name` is output-only fields 😄 
https://cloud.google.com/config-connector/docs/reference/resource-docs/logging/logginglogmetric